sql/postgres/inspect.go

```diff
@@ -851,17 +855,18 @@ ORDER BY
 	t1.table_schema, t1.table_name
 `
 	tablesQueryArgs = `
-SELECT
+	SELECT
 	t1.table_schema,
 	t1.table_name,
-	pg_catalog.obj_description(t2.oid, 'pg_class') AS comment,
-	t3.partattrs AS partition_attrs,
-	t3.partstrat AS partition_strategy,
-	pg_get_expr(t3.partexprs, t3.partrelid) AS partition_exprs
+	pg_catalog.obj_description(t3.oid, 'pg_class') AS comment,
+	t4.partattrs AS partition_attrs,
+	t4.partstrat AS partition_strategy,
+	pg_get_expr(t4.partexprs, t4.partrelid) AS partition_exprs
```

Comment:
Why did you change this code?

Reply:
For the same reason I modified `tablesQuery` - to avoid using `reg_class`